app/src/main/java/com/example/drawingApp/MainActivity.kt

```diff
@@ -169,6 +172,10 @@ class MainActivity : AppCompatActivity() {
             model.layerListClicked()
         }
 
+        findViewById<LinearLayout>(R.id.tabSheet).let { view ->
+            DialogUtility.tabSheetDialog(view, this)
```

Comment:
It doesn't look like this state is held anywhere. On rotation will this persist?

Reply:
no, fixed